test/settings/mysql.js

```diff
@@ -4,11 +4,11 @@ module.exports = {
   connection: {
     host     : 'localhost',
     user     : 'root',
-    password : 'numtel',
```

Comment:
This one...hmmm maybe put the comma at the beginning of the next line and add a comment to that `debug` will output all the incoming packets.